src/backend/main/main.py

```diff
@@ -49,4 +49,7 @@ async def check_server_status(request: Request):
 # Run this file to start the server
 if __name__ == "__main__":
     import uvicorn
-    uvicorn.run(main_app, host="0.0.0.0", port=8000, reload=True)
+    import os
+
+    port = int(os.environ.get("CHAT_SERVICE_PORT"))
```

Comment:
rename as chat_service_port

Reply:
I renamed this to chat_service_port in the code.